### DISCUSSION TITLE
[Feature request] Aliases folders

### DISCUSSION THREAD

**Discussion by leonardo-dgs** [anonaddy/anonaddy#446] [Feature Requests]
Folders will help to organize aliases into categories, instead of having one huge list of aliases.

**Comment by willbrowningme** [comment]
Thanks, that is a good idea and it's currently on the [roadmap](https://github.com/anonaddy/anonaddy/projects/1#card-52584414). Hopefully I can get time to add it soon.

**Reply by rehanone** [reply]
May I suggest a slightly different approach to this. Instead of folders which will make alias belong to just one folder, can we use labels? This will allow multiple labels to be applied to any alias and we can have a filter by alias to see a particular group of aliases.

**Reply by roke-julian-lockhart** [reply]
@rehanone, they're not mutually exclusive. Create a https://github.com/anonaddy/anonaddy/issues/new/choose for tag support if you want that (if an https://github.com/anonaddy/anonaddy/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc doesn't already exist). https://github.com/anonaddy/anonaddy/discussions/458#discussioncomment-5970589 exists though, and is about as good as an issue.

**Comment by willbrowningme** [comment]
Yes, labels / tags might be the best way to go about it.

**Comment by leonardo-dgs** [comment]
Maybe can be added both label and folders.

**Comment by ASBaumgarten** [comment]
Labels, tags, folder, color coding. Anything would be helpful.
